exporter/exporter.go

```diff
@@ -70,6 +70,9 @@ type Opts struct {
 	EnableCollStats          bool
 	EnableProfile            bool
 	EnableShards             bool
+	EnableFCV                bool // Feature Compatibility Version.
+
+	FCVInterval time.Duration
```

Comment:
I'm not sure if we need separate interval for FCV.

Reply:
this value changes only after upgrades so it should have a big interval to avoid unnecessary reads

Reply:
that's true, but it's not a good practice in exporter development. Is this operation generate a big load on system?

Reply:
no, not really. We can use one of the default collection intervals